### PR TITLE
Feature/nanoflow

### PR DESCRIPTION
**Pull request by jseremba**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -1,95 +1,164 @@
...
+        "always"
+      ],
+      "linebreak-style": [
+        true,
+        "LF"
+      ]
     }
-}
+  }
+
```
Comment (McDoyen) [inline tslint.json]: Missing end of line

File: tsconfig.json [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
     },
     "exclude": [
-        "dist/",
-        "node_modules/"
+      "dist/",
+      "node_modules/"
     ]
-}
+  }
+
```
Comment (McDoyen) [inline tsconfig.json]: Missing eol

**Review by McDoyen — CHANGES_REQUESTED** [review]

File: tslint.json (hunk 1 above)
Comment (jseremba) [reply tslint.json]: Done

File: tsconfig.json (hunk 2 above)
Comment (jseremba) [reply tsconfig.json]: Done

File: src/components/ProgressCircleContainer.ts [hunk 3]
```diff
@@ -22,6 +22,7 @@ export interface ContainerProps extends WrapperProps {
     positiveValueColor: BootstrapStyle;
     displayText: DisplayText;
     textSize: ProgressTextSize;
+    openPageAs: PageLocation;
```
Comment (djkwagala) [inline src/components/ProgressCircleContainer.ts:25]: use `OpenPageAs` typing

File: src/components/ProgressCircleContainer.ts [hunk 4]
```diff
@@ -38,6 +39,7 @@ interface ContainerState {
 
 export type DisplayText = "none" | "value" | "percentage";
 type OnClickOptions = "doNothing" | "showPage" | "callMicroflow" | "callNanoflow";
+type PageLocation = "popup" | "modal" | "content";
```
Comment (djkwagala) [inline src/components/ProgressCircleContainer.ts]: rename to `OpenPage` or `DisplayPage`

**Review by djkwagala — CHANGES_REQUESTED** [review]

**Pull request merged**